src/components/donation/Steps/common/Summary.tsx

```diff
@@ -51,29 +54,43 @@ export default function Summary({
             : "Total donation"}
         </dt>
         <Amount amount={props.amount} classes="text-navy-d4" />
-        <div className="flex items-center justify-between col-span-full">
-          <dt className="mr-auto text-sm">Sustainability Fund</dt>
-          <Amount classes="text-sm" amount={locked} />
-        </div>
-        <div className="flex items-center justify-between col-span-full">
-          <dt className="mr-auto text-sm">Direct Donation</dt>
-          <Amount classes="text-sm" amount={liq} />
-        </div>
-        {props.tip && (
+        {locked > 0 && (
+          <>
+            <div className="flex items-center justify-between col-span-full">
+              <div className="mr-auto flex">
+                <dt className="text-sm mt-2">Sustainability Fund</dt>
+                <Image src={character} className="inline-block px-1 h-8" />
+              </div>
+              <Amount classes="text-sm" amount={locked} />
+            </div>
+            <div className="flex items-center justify-between col-span-full">
+              <dt className="mr-auto text-sm">Direct Donation</dt>
+              <Amount classes="text-sm" amount={liq} />
+            </div>
+          </>
+        )}
+        {props.tip && props.tip.value > 0 && (
           <div className="col-span-full grid grid-cols-[1fr_auto] border-y border-gray-l4 py-3">
             <dt className="mr-auto">Donation for Better Giving</dt>
             <Amount classes="text-sm" amount={props.tip.value} />
           </div>
         )}
-        {props.tip && (
```

Comment:
the reason a separate `total` row is shown only when there's tip is that collation if only sensible if you have two distinct rows (og & bg)

<img width="748" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/4bcb203a-f913-4dc3-93ac-5e2a20e32469">

so for donations that doesn't have tip, the UI is redundant
<img width="760" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/06070692-595e-4337-84cb-a314182dccbf">

Reply:
Per your other comment, if we always show split breakdown then this scenario can'tt happen. I think this is prob what we can do as the best balance of things.

Reply:
redundant `Total` is unncessary imo as there's only one item (even if that item's detail/partition is enumerated)
<img width="759" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/fd0efdd4-2664-423d-adda-f256221750c4">

Reply:
It's a balance between not showing the breakdown or having it, but having no tip means it's restating the first line. It's like buying a VAT free product in the store. The subtotal and total are the same numbers. One thing the second total line adds is the frequency with which the donation will occur (monthly/one-time). 

I think we can freeze this PR scope for now for review/merge and refine further, only once we know what other chagnes will be needed as a separate ticket.

Reply:
> It's like buying a VAT free product in the store. The subtotal and total are the same numbers. 

makes sense